tf-managed/modules/daily-snapshot/variable.tf

```diff
@@ -109,6 +109,18 @@ variable "environment" {
   type        = string
 }
 
+
+variable "snapshot_type" {
+  type        = set(string)
+  description = "Set of network types for snapshot generation. Valid options: 'mainnet' and/or 'calibnet'"
+  default     = ["mainnet", "calibnet"]
+
+  validation {
+    condition     = alltrue([for val in var.snapshot_type : contains(["mainnet", "calibnet"], val)])
```

Comment:
Can we also check that the list is not empty?

Reply:
@LesnyRumcajs. done, please re-approve.